modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/OnboardingUpgradeHelper.kt

```diff
@@ -68,40 +73,50 @@ object OnboardingUpgradeHelper {
     @Composable
     fun UpgradeRowButton(
         primaryText: String,
-        backgroundColor: Color,
         textColor: Color,
         onClick: () -> Unit,
         modifier: Modifier = Modifier,
+        backgroundColor: Color = Color.Transparent,
         fontWeight: FontWeight = FontWeight.W600,
         secondaryText: String? = null,
+        gradientBackgroundColor: Brush? = null,
```

Comment:
I think it would be better to expose this as a single property `backgroundBrush: Brush`. Right now API is confusing because you can set two properties. Or if you want to have a simpler color access make two functions. One that exposes `Color` and the other that exposes `Brush`.

Reply:
You are right! I broke into two methods and added one more preview